stac_model/torch/export.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import tempfile
+from collections.abc import Sequence
+
+import torch
+import yaml
+from torch.export.dynamic_shapes import Dim
+from torch.export.pt2_archive._package import package_pt2
+
+from .utils import aoti_compile_and_extract, create_example_input_from_shape, extract_module_arg_names
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+def aoti_compile(
+    model_directory: str,
+    model_program: torch.export.ExportedProgram,
+    transforms_directory: str | None = None,
+    transforms_program: torch.export.ExportedProgram | None = None,
+) -> dict[str, list[str]]:
```

Comment:
Should consider a `TypedDict` and nested `list[os.PathLike[str]]` to be more explicit about the output.

Reply:
Added types PTAL